### ISSUE TITLE
Anti Ban Bypass

### ISSUE THREAD

**Issue by WeLoveGaming** [PoptartWeScript/WeScript#1]
Does this have anti ban bypass and can you please give a built version? I get errors when trying to build.

**Comment by PoptartWeScript** [comment]
yeah fuck you for trying to steal my code dumbass xDDDDDDDDDDDDDD
